eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -153,120 +151,113 @@ stages:
-                          - checkout: azure-sdk-build-tools
-                            path: azure-sdk-build-tools
-                          - template: /pipelines/steps/publish-symbols.yml@azure-sdk-build-tools
-                            parameters:
-                              BuildToolsRepositoryPath: $(Pipeline.Workspace)/azure-sdk-build-tools
-                              PackagesPath: $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed/${{artifact.name}}
-                              StagingDirectory: $(Build.ArtifactStagingDirectory)/symbols
-                          - template: /eng/common/pipelines/templates/steps/create-apireview.yml
-                            parameters:
-                              ArtifactPath: $(Pipeline.Workspace)/packages
-                              Artifacts: ${{parameters.Artifacts}}
-                              ConfigFileDir: $(Pipeline.Workspace)/packages/PackageInfo
-                              MarkPackageAsShipped: true
-                              ArtifactName: ${{parameters.ArtifactName}}
-                              PackageName: ${{artifact.name}}
+                          artifactName: ${{artifact.name}}-symbols
+
+                  steps:
+                    - checkout: self
+                      fetchDepth: 1
+                    - checkout: azure-sdk-build-tools
+                    - download: current
+                      artifact: ${{parameters.ArtifactName}}-signed
+                    - template: /pipelines/steps/publish-symbols.yml@azure-sdk-build-tools
+                      parameters:
+                        BuildToolsRepositoryPath: $(Pipeline.Workspace)/s/azure-sdk-build-tools
+                        PackagesPath: $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed/${{artifact.name}}
+                        StagingDirectory: $(Build.ArtifactStagingDirectory)/symbols
+                    - template: /eng/common/pipelines/templates/steps/create-apireview.yml
+                      parameters:
+                        ArtifactPath: $(Pipeline.Workspace)/packages
+                        Artifacts: ${{parameters.Artifacts}}
+                        ConfigFileDir: $(Pipeline.Workspace)/packages/PackageInfo
+                        MarkPackageAsShipped: true
+                        ArtifactName: ${{parameters.ArtifactName}}
+                        PackageName: ${{artifact.name}}
+                        SourceRootPath: $(Pipeline.Workspace)/s/azure-sdk-for-net
 
               - ${{if ne(artifact.skipPublishDocMs, 'true')}}:
-                - deployment: PublicDocsMS
+                - job: PublicDocsMS
                   displayName: Publish to Docs.MS
                   condition: and(succeeded(), ne(variables['Skip.PublishDocs'], 'true'))
-                  environment: package-publish
                   dependsOn: PublishPackage
 
                   pool:
                     name: azsdk-pool-mms-ubuntu-2004-general
```

Comment:
As a follow-up use image.yml variables for these.

Reply:
Had more yaml work to do, so added in this PR